server.py

```diff
@@ -40,20 +42,50 @@
 from lib.utils import validate_url
 from lib.utils import is_demo_node
 
-from settings import auth_basic, CONFIGURABLE_SETTINGS, DEFAULTS, LISTEN, PORT, settings, ZmqPublisher
+from settings import auth_basic, auth_system, CONFIGURABLE_SETTINGS, DEFAULTS, LISTEN, PORT, settings, ZmqPublisher
 
 HOME = getenv('HOME', '/home/pi')
 DISABLE_MANAGE_NETWORK = '.screenly/disable_manage_network'
+CELERY_RESULT_BACKEND = getenv('CELERY_RESULT_BACKEND', 'redis://localhost:6379/0')
```

Comment:
Is there a scenario where `CELERY_RESULT_BACKEND` and `CELERY_BROKER_URL` are different? If not, I suggest we consolidate them to `CELERY_BROKER`

Reply:
Hmm I think that variables likely will not different, but the result and broker have a different purposes. So I think we don't need to merger them into one variable, it can be confusing.